modules/data-table/main/index.coffee

```diff
@@ -786,10 +786,18 @@ urlFeed = (url, options) ->
 
   {
     url: url # for debugging
-    headers: maybeCached (r) -> hx.json(url, { type: 'headers', extra: options.extra }, r)
-    totalCount: maybeCached (r) -> hx.json(url, { type: 'totalCount', extra: options.extra }, (res) -> r(res.count))
-    rows: (range, cb) -> hx.json(url, { type: 'rows', range: range, extra: options.extra }, cb)
-    rowsForIds: (ids, lookupRow, cb) -> hx.json(url, { type: 'rowsForIds', ids: ids, extra: options.extra }, cb)
+    headers: maybeCached (cb) ->
+      hx.json url, { type: 'headers', extra: options.extra }, (err, res) ->
+        cb(res)
```

Comment:
This discards err without logging it! I'm not sure I like this.

I see a number of options -

     console.error(er) if er 
     throw er if er # might propogate to top of stack trace, or have other weird unexpected results
     hx.notify.negative("Oops! Something went wrong.") # very intrusive, although not unlike what the preference dialogue does.